src/support/index.coffee

```diff
@@ -105,7 +105,10 @@ module.exports = ->
     argv['prevent-browser-reload']?
 
   terminateDriver = =>
-    @driver.close()
-    @driver.quit()
+    if @driver?
+      @driver.close()
+      @driver.quit()
+    else
+      $.defer()
```

Comment:
shouldn't we return just a resolved promise here instead of a defered object that is in a non resolved state?